docs-source/spring/content/security/azn-server/_index.md

```diff
@@ -0,0 +1,136 @@
+
+| User Name     | Assigned Roles        |
+|---------------|-----------------------|
+| obaas-admin   | ROLE_ADMIN, ROLE_USER |
+| obaas-user    | ROLE_USER             |
+
+All the users are stored in the database deployed when installing Oracle Backend for Spring Boot and Microservices. The Roles determines what the user is allowed to do in the environment. The allowed roles are `ROLE_ADMIN` and `ROLE_USER`.
+
+> **_NOTE:_** See each components documentation about the roles and authorities.
+
+The assigned passwords (either auto generated or provided by the installer) can be viewed in the OCI Console (ORM homepage). Click on **Application Information**.
+
+<!-- spellchecker-disable -->
+{{< img name="azn-stack-app-info" >}}
+<!-- spellchecker-enable -->
+
+If you click on **Unlock** the password for the obaas-admin and obaas-user can be displayed.
+
+<!-- spellchecker-disable -->
+{{< img name="azn-passwords-console" >}}
+<!-- spellchecker-enable -->
+
+The passwords can also be obtained from k8s secrets using the `kubectl` command.
+
+For `obaas-admin`:
+
+```shell
+kubectl get secret -n azn-server oractl-passwords -o jsonpath='{.data.admin}' | base64 -d
+```
+
+For `obaas-user`:
+
+```shell
+kubectl get secret -n azn-server oractl-passwords -o jsonpath='{.data.user}' | base64 -d
+```
+
+## User Management REST endpoints
+
+The following REST endpoints are available to manage users. The table lists which minimum required Role is needed to perform the operation.
+
+| End point                                         | Method | Description                                     | Minimum required Role |
+|---------------------------------------------------|--------|-------------------------------------------------|-----------------------|
+| /user/api/v1/findUser                             | GET    | Find all users                                  | ROLE_ADMIN            |
+| /user/api/v1/findUser?username=\<username\>       | GET    | Find a user with the username \<username\>      | ROLE_ADMIN            |
+| /user/api/v1/createUser                           | POST   | Create a user                                   | ROLE_ADMIN            |
+| /user/api/v1/updatePassword                       | PUT    | Update a password for a user. A user with<br>Role ROLE_ADMIN can update any users password | ROLE_USER |
+| /user/api/v1/deleteUsername?username=\<username\> | DELETE | Delete a user with username \<username\>        | ROLE_ADMIN            |
+| /user/api/v1/deleteId?id=\<id\>                   | DELETE | Delete a user with the id \<id\>                | ROLE_ADMIN            |
+
+### User Management REST Endpoints
+
+In all examples below you need to replace `<username>:<password>` with your username and password. The examples are using `curl` to interact with the REST endpoints. They also requires that you have opened a tunnel on port 8080 to either the `azn-server` or `obaas-admin` service. For example
```

Comment:
First sentence: "In all of the following examples, you ..."
Third sentence: "They also require that you ..."
Add a : after "example"